pkg/resourcecreator/deployment.go

```diff
@@ -472,6 +472,14 @@ func podObjectMeta(app *nais.Application) metav1.ObjectMeta {
 		objectMeta.Labels["kafka"] = "enabled"
 	}
 
+	if (app.Spec.Tracing != nil && app.Spec.Tracing.Enabled) {
+		objectMeta.Annotations["proxy.istio.io/config"] =
```

Comment:
Her kan det se bedre ut med multiline string

Reply:
Selv om det må bryte med indentation i koden for å være gyldig?

`        if (app.Spec.Tracing != nil && app.Spec.Tracing.Enabled) {
                objectMeta.Annotations["proxy.istio.io/config"] = `tracing:
  sampling: 0
  zipkin:
    address: zipkin.istio-system:9411`
        }`

Reply:
Går an å gjøre det til en konstant i stedet, da blir koden fin og ryddig.